### DISCUSSION TITLE
SSH Freezes on remote IP Change using netsh

### DISCUSSION THREAD

**Discussion by prasad-dash** [PowerShell/Win32-OpenSSH#2173] [Q&A]
I am trying to change the IP of VM from local machine using ssh and passing 'ssh xyz@11.1.1.90 'netsh interface ipv4 set address "Ethernet0" static 11.1.1.6 255.255.255.0' ' . The systems are configured with key based authentication. The IPv4 addresses mentioned are custom based on a project. The IP changes successfully but the powershell freezes after that and  on checking debug statements its is revealed that control is only returned after connection timeout.  
Version of ssh -OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2
Version of sshd on VM -OpenSSH_for_Windows_7.7p1, LibreSSL 2.6.5
PSVersionTable output Name                           Value
----                           -----
PSVersion                      5.1.19041.3570
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.3570
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

I suspect that the connection between the systems is broken as the ip is changed but the same is not informed to the host. Can someone clarify and help solve it.

**Comment by maertendMSFT** [comment]
@prasad-dash, the disconnect in this scenario makes sense and is by design.  The question that we have is what is your desired resolution after the disconnect occurs?  Are you looking for the connection to close immediately, but instead you are having to wait for the timeout?
